Signal/src/view controllers/SignalsViewController.m

```diff
@@ -150,22 +158,71 @@ - (UIViewController *)previewingContext:(id<UIViewControllerPreviewing>)previewi
     }
 }
 
-- (void)handleActiveCallNotification:(NSNotification *)notification
+- (void)handlePresentCallInterstitialNotification:(NSNotification *)notification
 {
     AssertIsOnMainThread();
+    
+    NSString *callToken = notification.object;
+    OWSAssert(callToken != nil);
+    
+    OWSCallInterstitialViewController *viewController = [OWSCallInterstitialViewController new];
+    viewController.callToken = callToken;
```

Comment:
nit: As I understand it, every `CallInterstitialViewController` needs a `callToken`. 

Rather than setting it right here, and `assert`ing it's presence in CallInterstitialViewController, my impulse would be to make `callToken` a `let` and pass it to a `required` initializer - offloading some of this contract to language semantics.

e.g.  `[[OWSCallInterstitialViewController alloc] initWithCallToken:callToken];`